src/components/calendarpage.jsx

```diff
@@ -194,9 +223,10 @@ export function CalendarPage() {
         selectable={true}
         plugins={[dayGridPlugin, interactionPlugin]}
         dateClick={handleDateClick}
-        eventClick={openModalFromCalendar}
+        eventClick={openModalFromCalendar} // TODO: Display a different modal
```

Comment:
What's with the different modal? And do we need to do it before merging in this PR?

Reply:
Updated TODO: `TODO: Fix existing modal to support editing period data or display a different modal`